v1.1/badge/open-badge-credential.descriptor.json

```diff
@@ -71,6 +71,26 @@
         "type": "string"
       }
     },
+    {
+      "label": "Stare time",
```

Comment:
`Start time` or i'd prefer simpler `Start` (since its a date and time)

Reply:
Fixed. Please approve @sloops77